mlflow/langchain/langchain_tracer.py

```diff
@@ -36,41 +37,33 @@ class MlflowLangchainTracer(BaseCallbackHandler, metaclass=ExceptionSafeAbstract
     input arguments added to original function call.
 
     Args:
-        parent_span: Optional parent span for the trace. If provided, spans will be
-            created under the given parent span. Otherwise, a single trace will be
-            created. Example usage:
-
-            .. code-block:: python
-
-                from mlflow.langchain.langchain_tracer import MlflowLangchainTracer
-                from langchain_community.chat_models import ChatDatabricks
-
-                chat_model = ChatDatabricks(endpoint="databricks-llama-2-70b-chat")
-                with mlflow.start_span("Custom root span") as root_span:
-                    chat_model.invoke(
-                        "What is MLflow?",
-                        config={"callbacks": [MlflowLangchainTracer(root_span)]},
-                    )
         prediction_context: Optional prediction context object to be set for the
             thread-local context. Occasionally this has to be passed manually because
             the callback may be invoked asynchronously and Langchain doesn't correctly
             propagate the thread-local context.
+        set_span_in_context: If True, the span created by this callback will be set as
+            the active span and attached to the current context. This will allow using
+            fluent APIs or other auto-tracing integrations with this callback. If set
+            to False, the span will not be set as active span.
     """
 
     def __init__(
-        self, parent_span: Optional[LiveSpan] = None, prediction_context: Optional[Context] = None
+        self,
+        prediction_context: Optional[Context] = None,
+        set_span_in_context: bool = True,
     ):
         # NB: The tracer can handle multiple traces in parallel under multi-threading scenarios.
         # DO NOT use instance variables to manage the state of single trace.
         super().__init__()
         self._mlflow_client = MlflowClient()
-        self._parent_span = parent_span
-        self._run_span_mapping: dict[str, LiveSpan] = {}
-        self._active_request_ids: set[str] = set()
+        # run_id: (LiveSpan, OTel token)
```

Comment:
Question: what is `Tel token`?

Reply:
`OTel token` = A contextvar token object that is generated when setting a span as active within the Open Telemetry span context. This token is required when inactivate the span i.e. detaching the span from the context.